src/webui/www/private/scripts/torrent-content.js

```diff
@@ -0,0 +1,572 @@
+            const remaining = (row.priority === FilePriority.Ignored) ? 0 : row.remaining;
+            const childNode = new window.qBittorrent.FileTree.FileNode();
+            childNode.name = row.name;
+            childNode.path = row.fileName;
+            childNode.rowId = rowId;
+            childNode.size = row.size;
+            childNode.checked = isChecked;
+            childNode.remaining = remaining;
+            childNode.progress = row.progress;
+            childNode.priority = row.priority;
+            childNode.availability = row.availability;
+            childNode.root = parent;
+            childNode.data = row;
+            parent.addChild(childNode);
+
+            ++rowId;
+        });
+
+        torrentFilesTable.populateTable(rootNode);
+        torrentFilesTable.updateTable();
+
+        if (selectedFiles.length > 0)
+            torrentFilesTable.reselectRows(selectedFiles);
+    };
+
+    const filesPriorityMenuClicked = (priority) => {
+        const selectedRows = torrentFilesTable.selectedRowsIds();
+        if (selectedRows.length === 0)
+            return;
+
+        const rowIds = [];
+        const fileIds = [];
+        selectedRows.forEach((rowId) => {
+            rowIds.push(rowId);
+            fileIds.push(Number(torrentFilesTable.getRowFileId(rowId)));
+        });
+
+        const uniqueRowIds = new Set();
+        const uniqueFileIds = new Set();
+        for (let i = 0; i < rowIds.length; ++i) {
+            const rows = getAllChildren(rowIds[i], fileIds[i]);
+            rows.rowIds.forEach((rowId) => {
+                uniqueRowIds.add(rowId);
+            });
+            rows.fileIds.forEach((fileId) => {
+                uniqueFileIds.add(fileId);
+            });
+        }
+
+        setFilePriority([...uniqueRowIds.keys()], [...uniqueFileIds.keys()], priority);
+        for (const id of rowIds)
+            updateParentFolder(id);
+    };
+
+    const updateParentFolder = (id) => {
+        const updateComplete = () => {
+            // we've finished recursing
+            updateGlobalCheckbox();
+            torrentFilesTable.updateTable(true);
+        };
+
+        const node = torrentFilesTable.getNode(id);
+        const parent = node.parent;
+        if (parent === torrentFilesTable.getRoot()) {
+            updateComplete();
+            return;
+        }
+
+        const siblings = parent.children;
+
+        let checkedCount = 0;
+        let uncheckedCount = 0;
+        let indeterminateCount = 0;
+        let desiredComboboxPriority = null;
+        for (const sibling of siblings) {
+            switch (getCheckboxState(sibling.rowId)) {
+                case TriState.Checked:
+                    checkedCount++;
+                    break;
+                case TriState.Unchecked:
+                    uncheckedCount++;
+                    break;
+                case TriState.Partial:
+                    indeterminateCount++;
```

Comment:
```suggestion
                    ++checkedCount;
                    break;
                case TriState.Unchecked:
                    ++uncheckedCount;
                    break;
                case TriState.Partial:
                    ++indeterminateCount;
```

Reply:
This code was moved and is unchanged. Will leave modifications to a separate PR

Reply:
>This code was moved and is unchanged.

Could you point out where was the code in master?

Reply:
I was mistaken, this is new code. Will change.